From d663be1a54d52be58cdc2241e032890aa6ad137c Mon Sep 17 00:00:00 2001
Message-ID: <d663be1a54d52be58cdc2241e032890aa6ad137c.1699403992.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:55:25 +0100
Subject: [PATCH 19/67] gnu: go-github-com-jcmturner-goidentity-v6: Move to
 (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-jcmturner-goidentity-v6): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Ibd9202efe7fc1002bc34376107a0e88872c888be
---
 gnu/packages/golang-web.scm | 24 ++++++++++++++++++++++++
 gnu/packages/golang.scm     | 24 ------------------------
 2 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 6a832291ab..751eb41b58 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -314,6 +314,30 @@ (define-public go-github-com-jcmturner-dnsutils-v2
 network applications that require accessing services using SRV records.")
     (license license:asl2.0)))
 
+(define-public go-github-com-jcmturner-goidentity-v6
+  (package
+    (name "go-github-com-jcmturner-goidentity-v6")
+    (version "6.0.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/jcmturner/goidentity")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "064ysvxvrvij843s7qj1nkzl5qc6j1qbrsb3s0zmwd1sa7vq8q1n"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/jcmturner/goidentity/v6"))
+    (propagated-inputs
+     (list go-github-com-stretchr-testify go-github-com-hashicorp-go-uuid))
+    (home-page "https://github.com/jcmturner/goidentity")
+    (synopsis "Hold authenticated identities and their attributes")
+    (description "This package provides a standard interface for holding
+authenticated identities and their attributes.")
+    (license license:asl2.0)))
+
 (define-public go-github-com-jcmturner-gokrb5-v8
   (package
     (name "go-github-com-jcmturner-gokrb5-v8")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 20d3c378c4..7ad906ef54 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1362,30 +1362,6 @@ (define-public go-github-com-jcmturner-aescts-v2
 encryption and decryption methods.")
     (license license:asl2.0)))
 
-(define-public go-github-com-jcmturner-goidentity-v6
-  (package
-    (name "go-github-com-jcmturner-goidentity-v6")
-    (version "6.0.1")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/jcmturner/goidentity")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "064ysvxvrvij843s7qj1nkzl5qc6j1qbrsb3s0zmwd1sa7vq8q1n"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/jcmturner/goidentity/v6"))
-    (propagated-inputs
-     (list go-github-com-stretchr-testify go-github-com-hashicorp-go-uuid))
-    (home-page "https://github.com/jcmturner/goidentity")
-    (synopsis "Hold authenticated identities and their attributes")
-    (description "This package provides a standard interface for holding
-authenticated identities and their attributes.")
-    (license license:asl2.0)))
-
 (define-public go-github-com-jcmturner-gofork
   (package
     (name "go-github-com-jcmturner-gofork")
-- 
2.41.0

